README.md

```diff
@@ -2,101 +2,67 @@
 
 Fork of [paritytech/polkadot-testnet-faucet](https://github.com/paritytech/polkadot-testnet-faucet)
 
-## Development
+E2E tests don't work right now, and the fork has diverged significantly and stopped being synched around v3.4
 
-#### Setup dependencies and git hooks
+To test, follow instructions to launch the Faucet Server, then head over to the Faucet Client README and launch that.
 
+## Faucet Client Development
+See [Faucet Client README](https://github.com/frequency-chain/testnet-faucet/blob/main/client/README.md)
+
+## Faucet Server Development
+### 1. Install packages.
 ```bash
 yarn install
-yarn simple-git-hooks
 ```
 
-#### start local database:
+### 2. start local database:
 
 Uses sqlite file, no start needed.
 
-#### run migrations:
+### 3. run migrations:
 ```bash
 yarn migrations:run
 ```
 
-#### creating migrations:
-* update entities in `src/db/entity`
-* run `yarn migrations:generate src/db/migration/<migration_name>`
-* import generated migration to `src/db/dataSource.ts`
-
-#### To launch a hot-reloading dev environment
-
-```bash
-yarn dev
-```
-
-## Environment variables
+### 4. Configure environment variables
 
 Definition with explanation is in `./env.faucet.config.json`
 
 Copy example file to real env and change its values:
 ```bash
-$ cp example.env .env
+$ cp .env.example .env
 ```
 
-## End-to-end tests
+### 5. launch a hot-reloading dev environment (faucet server only)
 
-Please refer to the [E2E Readme](./e2e/README.md).
+```bash
+yarn dev
```

Comment:
we're pretty consistently using `npm run dev` across projects. why are we switching to yarn?

Reply:
Probably out of scope for this story but just curious.

Reply:
The faucet already was using yarn, so there was no switching involved.  Switching to npm would be out of scope, yes.